server/issue.go

```diff
@@ -26,8 +27,9 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	api := ji.GetPlugin().API
 
 	create := &struct {
-		PostId string           `json:"post_id"`
-		Fields jira.IssueFields `json:"fields"`
+		RequiredFieldsNotCovered []string         `json:"required_fields_not_covered"`
```

Comment:
Empty array if all required fields are covered by JiraFields.  Else we are going to respond with MM post with link to manually create Jira Ticket (prefilled with fields that were entered by user)

Reply:
I'd be wary of the client code relying on an empty array vs null. Will look there.

Reply:
@lev Let me clarify and see if this answers your concern.  

`RequiredFieldsNotCovered` will be an empty array, in the event all Required fields were covered by our code.  If some required fields aren't covered, this array will be filled with the names of the fields and will be displayed back the user in an ephemeral post.

Does clear up your concern or am I still missing the issue?

Reply:
The question was, really, why {}, not `nil`?